interface/forms/vitals/templates/vitals/vitals_textbox_conversion.tpl

```diff
@@ -0,0 +1,75 @@
+<!-- USA row comes first -->
+    {if $units_of_measurement == $MEASUREMENT_METRIC_ONLY}
+    <tr class="hide">
+    {else}
+    <tr>
+    {/if}
+    {if $units_of_measurement == $MEASUREMENT_PERSIST_IN_METRIC}
+        <td class="unfocus graph" id="{$input|attr}">
+    {else}
+        <td class="graph" id="{$input|attr}">
+    {/if}
+            {xlt t=$title}
+        </td>
+    {if $units_of_measurement == $MEASUREMENT_PERSIST_IN_METRIC}
+        <td class="unfocus">
+    {else}
+        <td>
+    {/if}
+            {xlt t=$unit}
+        </td>
+    {if $units_of_measurement == $MEASUREMENT_PERSIST_IN_METRIC}
+        <td class="valuesunfocus">
+    {else}
+        <td class='currentvalues p-2'>
+    {/if}
+            <input type="text" class="form-control" size='5' name='{$input|attr}' id='{$input|attr}_input'
+                   value="{if $vitals->$vitalsValue() != 0}{$vitals->$vitalsValue()|attr}{/if}"
+                   onChange="convUnit('usa', '{$unit}','{$input|attr}_input')" title='{$vitalsValueUSAHelpTitle|default:''|xlt}'/>
```

Comment:
tested following line to work:
```
onChange="convUnit('usa', {$unit|attr_js}, '{$input|attr}_input')" title='{$vitalsValueUSAHelpTitle|default:''|xlt}'/>
```
yep, pretty much obsessed with html escaping :)
(note need to clear the smarty cache in Administration->Clinic->Calendar after make this change)